src/index.js

```diff
@@ -80,7 +78,9 @@ server.get("/dnslink/:name", async (req, res) => {
     console.log(`${req.params.name} => ${skylink}`);
 
     return success({ skylink });
-  });
+  }
+
+  return failure(`No TXT record found for _dnslink.${req.params.name}`);
```

Comment:
There is a number of specific user facing errors that have been squashed as this generic one, degrading logs potential debugging usability and less informative for end user that tries to set up dnslink on a domain.